lib/postcode_validation/gateway/pca_address_list.rb

```diff
@@ -0,0 +1,43 @@
+module PostcodeValidation
+  module Gateway
+    class PCAAddressList
+      include HTTParty
+
+      class PCARequestError < PostcodeValidation::Error::RequestError; end
+
+      KEY = ENV['POSTCODE_ANYWHERE_KEY']
+      base_uri 'https://services.postcodeanywhere.co.uk'
```

Comment:
Just tried to extract the duplication into a mixin, but it has trouble with the self.class.get and other stuff.